setup.cfg

```diff
@@ -117,6 +117,9 @@ install_requires =
     markupsafe>=1.1.1
     marshmallow-oneofschema>=2.0.1
     mdit-py-plugins>=0.3.0
+    # Pip can not find a version that satisfies constraints if opentelemetry-api is not pinned.
+    opentelemetry-api==1.15.0
```

Comment:
I believe you should be able to replace it with`>= 1.15.0` now. The problems were likely in Python 3.7 only and caused by importlib_metada incompatibilities. Those will be finally solved when Celery releases 5.3 (they have rc1 as of 11 May).

https://pypi.org/project/celery/#history

Reply:
Sweet, I'll test it and see if it works now.  I had missed that update, thanks.

Reply:
Seems to still run into the issue when I use `>=`.  `breeze ci-image build --upgrade-to-newer-dependencies --python 3.8` is still running after 2500s